napari/__init__.py

```diff
@@ -1,18 +1,12 @@
-from ._version import get_versions
-
-__version__ = get_versions()['version']
-del get_versions
+try:
+    from ._version import version as __version__
+except ImportError:
+    __version__ = "unknown"
```

Comment:
What exactly is happening here? I very much dislike 'unknown' as our version, which I don't think ever happened with versioneer?

Reply:
I forget at the moment exactly what versioneer shows for __version__ when it can't find the tags, but it's also capable of failure. 

the `_version` file will _only_ be unavailable if someone has downloaded the source tree from github, and _not_ installed it with `pip install -e .` (in which case, they basically won't be able to import anything anyway, so it's a very rare case).  However, it gets added when we run `python setup.py sdist bdist_wheel`, so it will always be in our distributions, and it also gets run with `pip install -e .`, so it will be there with local editable installs.

Reply:
Hmm. Ok, so the next question is, do we want to make this an editable string that we maintain on each release, skimage style? So now it would read `0.3.4.dev0`. I'm gonna guess the answer is no, but I'd like to have the discussion anyway. 'unknown' is very unsatisfying to me.

Another option that doesn't require ongoing maintenance might be `'not-installed'`?

Reply:
I'm not opposed to either!  I think `not-installed` is self-descriptive and requires no maintenance.  unless @sofroniewn wants to add "update that string" to the release checklist?

Reply:
will make it "not-installed" now, and we can update it later with `.dev0` if we go that way